source/dev/idea_list.md

```diff
@@ -0,0 +1,147 @@
+# Google Summer of Code 2021
+Breakthrough Energy Sciences (BES) has applied to serve as a mentoring organization for Google Summer of Code (GSoC).
+The GSoC program offers each student a stipend to work on open source code during the summer. 
+If you are interested in the program [here](https://summerofcode.withgoogle.com/) you can find more information.
+
+If you are a student and new to GSoC, please read the following [guide](https://google.github.io/gsocguides/student/).
+If you are new to BES, look [here](https://science.breakthroughenergy.org/) to learn more about our organization.
+
+
+## Community and Communication
+We are active on Slack and this is the best way to reach us.
+[Join us here](http://science.breakthroughenergy.org/#get-updates), introduce yourself, discuss your potential project in the #ideas channel, and let us know if you have any questions.
+
+If you need to reach us via email, here you go: sciences@breakthroughenergy.org
+
+Please review our [code of conduct](code_of_conduct.md) before joining the community.
+## Application
+If you are a student and applying for GSoC find bellow the idea list and explore the projects we would like to accomplish with you this summer.
+
+When applying please provide:
+
+- A project proposal in markdown following the [Elements of a Quality Proposal guide](https://google.github.io/gsocguides/student/writing-a-proposal#elements-of-a-quality-proposal)
+- Motivation why you choose this particular project?
+- Your resume with relevant experience in PDF format
+## Idea list
+If you can’t find your ideal project on this list,
+but you have something else in mind that would enhance our open source project,
+let's discuss it the #ideas channel on Slack.
+For each project we provided various scales to adapt to your skill, ambition or individual situation.
```

Comment:
For each project, we provide prerequisites along with a  level of difficulty that should assist you in your decision.

Reply:
Would be nice if somebody would have time for it before the deadline.